manifests/repo/rhel/erlang.pp

```diff
@@ -0,0 +1,23 @@
+# Makes sure that the Packagecloud repo for erlang is installed
+#
+# @api private
+class rabbitmq::repo::rhel::erlang (
+  $location          = "https://packagecloud.io/rabbitmq/erlang/el/${facts['os'][release][major]}/\$basearch",
+  String $key_source = $rabbitmq::erlang_package_gpg_key,
```

Comment:
```suggestion
  Stdlib::HTTPSUrl $key_source = $rabbitmq::erlang_package_gpg_key,
```

Reply:
this is passed to --import option of the rpm command which can take various types like https url, http url or file path so HTTPSUrl doesn't work.

Reply:
The same as the previous comment.